cfgov/jinja2/v1/_includes/organisms/info-unit-group-2.html

```diff
@@ -59,19 +64,28 @@
     {{ value.intro | safe }}
 {%- endif %}
 
-    <div class="content-l">
-    {% for unit in value.info_units %}
-        <div class="content-l_col
-                    {{- ' content-l_col-1-2' if value.format == '50-50' else '' -}}
-                    {{- ' content-l_col-1-3' if value.format == '33-33-33' else '' -}}
-                    {{- ' content-l_col-1' if value.format == '25-75' else '' -}}
-                   ">
-        {% if value.lines_between_items and value.format == '25-75' and loop.index > 1 %}
-            <div class="a-rule-break"></div>
-        {% endif %}
-            {{- info_unit(unit, value.format, value.link_image_and_heading) }}
-        </div>
-    {% endfor %}
+{% for unit in value.info_units %}
+    {% if ( loop.index == 1 ) or
+          ( value.format == '50-50' and loop.index % 2 == 1 ) or
+          ( value.format == '33-33-33' and loop.index  % 3 == 1 ) 
+    %}
+    <div class="content-l
+                {{' content-l__top-divider' if value.lines_between_items else ''}}">
+    {% endif %}
+    
+    <div class="content-l_col
+                {{- ' content-l_col-1-2' if value.format == '50-50' else '' -}}
+                {{- ' content-l_col-1-3' if value.format == '33-33-33' else '' -}}
+                {{- ' content-l_col-1' if value.format == '25-75' else '' -}}">
+        {{- info_unit(unit, value.format, value.link_image_and_heading) }}
     </div>
```

Comment:
Trivial: This `div` should be indented one more level

Reply:
Should be fixed in https://github.com/cfpb/cfgov-refresh/pull/3476/commits/9f21d322fc70c2ba1ebd623972e54df7c0087e47